lib/pop_db.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+var Skribbl = require("../models/skribbl.js");
+var lorem = require('lorem-ipsum');
+var popdb = require('./pop_db.js');
+
+exports.loremIpafy = function (skribblDict){
```

Comment:
Technically, its the same as a Dictionary, but JS terminology is more correct with Obj....